src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -239,8 +239,21 @@ def register_submission(
         )
 
         for attachment in submission.attachments:
+            # attribute names conform to the Open Documenten API specification
             iot = attachment.informatieobjecttype or options["informatieobjecttype"]
-            doc_options = {**options, "informatieobjecttype": iot}
+            bron = attachment.bronorganisatie or options["organisatie_rsin"]
```

Comment:
```suggestion
            bronorganisatie = attachment.bronorganisatie or options["organisatie_rsin"]
```
let's stick to the terms of the standard